lib/src/kvs/rocksdb/mod.rs

```diff
@@ -9,10 +9,11 @@ use rocksdb::IteratorMode;
 use rocksdb::OptimisticTransactionDB;
 use rocksdb::ReadOptions;
 use std::ops::Range;
+use std::pin::Pin;
 use std::sync::Arc;
 
 pub struct Datastore {
-	db: rocksdb::OptimisticTransactionDB,
+	db: Pin<Arc<rocksdb::OptimisticTransactionDB>>,
```

Comment:
`Pin`, not because `rustc` suggested it, but because that is the actual contract of this variable (it *cannot* move in memory)